libkineto/include/GenericTraceActivity.h

```diff
@@ -16,6 +16,15 @@
 #include "TraceActivity.h"
 #include "TraceSpan.h"
 
+#define GENERATE_FORWARD_BACKWARD_LINK(link_id, link_type) \
+  ((((uint64_t)(link_type)) << 60) | (link_id))
+#define GET_LINK_TYPE(link) \
+  (((uint64_t)(link)) >> 60)
+#define GET_LINK_ID(link) \
+  ((((uint64_t)1 << 60) - 1) & (link))
+#define LINK_FORWARD_BACKWARD_HEAD 1
```

Comment:
We can use one type here I think, just to determine category and name:
constexpr unsigned int kLinkFwdBwd = 1;